core/src/main/java/org/lflang/generator/docker/CDockerGenerator.java

```diff
@@ -40,7 +40,7 @@ public List<String> defaultEntryPoint() {
   @Override
   protected String generateRunForInstallingDeps() {
     var config = context.getTargetConfig();
-    var compiler = config.target == Target.CCPP ? "g++" : "gcc";
+    var compiler = config.target == Target.CCPP || config.target == Target.CPP ? "g++" : "gcc";
```

Comment:
```suggestion
    var compiler = config.target == Target.CCPP ? "g++" : "gcc";
```
(because `CppDockerGenerator` no longer extends `CDockerGenerator`